Packs/CertStream/Playbooks/playbook-Create_list_for_PTH.yml

```diff
@@ -745,43 +1043,224 @@ tasks:
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "43":
+    id: "43"
+    taskid: 53f636fa-3ec1-46a0-87f7-706780d17d23
+    type: regular
+    task:
+      id: 53f636fa-3ec1-46a0-87f7-706780d17d23
+      version: -1
+      name: Set ListData to Context
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.10/Cortex-XSOAR-Administrator-Guide/Automations
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "13"
+    scriptarguments:
+      append:
+        simple: "false"
+      key:
+        simple: ListData
+      value:
+        complex:
+          root: OpenAiChatGPTV3.Conversation.[0]
+          accessor: assistant
+          transformers:
+          - operator: StripChars
+            args:
+              chars:
+                value:
+                  simple: '{}'
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -140,
+          "y": 1230
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "44":
+    id: "44"
+    taskid: 0b41e822-f88a-4f5b-81c2-6a05ce46e3d5
     type: condition
     task:
-      id: a8535e2e-b09f-47d7-8ec5-bc075377a7a7
+      id: 0b41e822-f88a-4f5b-81c2-6a05ce46e3d5
       version: -1
-      name: Manual\Xpanse permutation
-      description: ''
+      name: List exist - should overwrite?
+      description: ""
       type: condition
       iscommand: false
       brand: ""
     nexttasks:
       '#default#':
-      - "7"
-      Use LLM Permutations:
-      - "10"
+      - "9"
+      "Yes":
+      - "5"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 660,
+          "y": -530
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+        simple: List with the name ${inputs.DomainsListName} already exist, should
+          we continue and overwrite it?
+      methods: []
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+      replyOptions:
+      - "Yes"
+      - "No"
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "45":
```

Comment:
Combine the two tasks (one asking for analyst decision and the other checking the answer) into one conditional ask task.